package.json

```diff
@@ -16,12 +16,13 @@
     "compile": "lerna run --stream compile",
     "lint": "lerna run lint",
     "publish": "node scripts/publish.js",
-    "test": "lerna run test --concurrency 1",
+    "test": "lerna run test --concurrency 1 --stream",
```

Comment:
@guw @GaryFNC @vazexqi I'm adding this flag to prevent travis timing out because of lack of output.

Reply:
Talked to Nick and holding off on this change for now.